source/BackwardTaintEnvironment.cpp

```diff
@@ -38,6 +40,24 @@ BackwardTaintEnvironment BackwardTaintEnvironment::initial(
             /* user_features */ {})));
   }
 
+  if (context.options.propagate_via_arg() && !context.previous_model.is_frozen(Model::FreezeKind::Propagations)) {
+    for (ParameterPosition i = method.first_parameter_index(); i < method.number_of_parameters(); ++i) {
+      const DexType* const argument_type = method.parameter_type(i);
+      if (type::is_object(argument_type)) {
```

Comment:
nit: prefer early return to avoid one nested level
```suggestion
      if (!type::is_object(argument_type)) {
        continue;
      }
```